### DISCUSSION TITLE
Fan control groupings

### DISCUSSION THREAD

**Discussion by vladdeda** [Rem0o/FanControl.Releases#1969] [Ideas]
I think it'd be awesome if we could add fans to a group ie. CPU and then set however many fans to that group, then have just 1 setting to control that group of fans. I have 18 case fans and 4 cpu fans in my PC and want them all to be controlled by the same thing. But for each fan I have to go in and change 22 settings. It'd be nice to just have to change 1 setting for the group and be done with it! What does everyone think?

**Comment by laleeroy** [comment]
That would be great, instead of setting a profile for each fan and renaming each fan.
It would also look neat, with less cards

**Comment by EvanMulawski** [comment]
I believe you can do this already using a Mix curve with only one curve selected. The "group" is the Mix curve. Set all fans you want in the group to use the Mix curve. You can then change all curves for the group by changing the curve selected in the Mix curve.

Edit: Just realized you probably mean the start/stop/step settings. I agree with you there.

**Comment by Rem0o** [comment]
Just to be clean, we are talking about grouping these settings?

![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/2d2849e8-a34c-4513-b58e-65d2de048f78)

Aren't those settings a single set and forget kind of thing which the assisted setup will fill in automatically anyway? Also, even if you use the same model of fan, the start/stop can change substantially from one fan to another. On my own setup I have a 10% difference on a set of fans for the start%.

**Reply by vladdeda** [reply]
No, I'm talking about this. 
![image](https://github.com/Rem0o/FanControl.Releases/assets/19613372/1a4f9507-3834-4333-ad1e-304d9210e797)

Making it so you could have group "CPU" with fans '1,2,3,4' then say another group 'GPU' with fans '5,6,7,8' and another group 'Front" with '9,10,11,12' and so on.... Then set each group to it's own fan curve or the same fan curve, whatever I wanted or needed for a certain scenario.

I have two fans listed there as my GPU, the 3080, it'd be nice to just flip one switch to turn on control for them as they'd be in the own group, and set a curve to both fans with one setting, instead of both fans individually. All of the fans on the corsair devices are all the same fans, so I don't have to worry about variations in their speeds and could set the ones I want (near the GPU) to push more air to the GPU with a specific fan curve pointed at the GPU temp all at once, instead of doing each fan individually. Right now I have some fans labeled, that's for when I get my GPU water cooled. I'd like to have those on a single group with their own curve. That functionality isn't there yet. That's what I'd like to see. :D

**Reply by Rem0o** [reply]
Have you tried using sync fan curves? Set a group of fans to sync with a specific control, then apply the wanted curve on that control and everything will sync together.

**Reply by vladdeda** [reply]
That's interesting, I'll have to look into that. I still like the idea of a grouping system where the fans could be put under one group. What you've described is as I see it a work around to grouping, and while it appears to work, it's still a lot to set all of these fans shown above to one fan. or numerous fans to one control any time something changes.

I've just tested this with all of my fans and not all of them respond to the sync'd command. idk if that's a software issue or what. I'm really trying to get away from ever having to boot up icue... XD

**Comment by laleeroy** [comment]
Im not a programmer so I don't know how hard it is to implement, but it would be cool if we could drag and drop one fan card to another card and combine them to become one group
